.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: PRISM Test
```

Comment:
Since there is another file running PRISM CI already exists, what we can do is split tests between the frontend and the backend, so maybe rename this file: `api_test.yml` and the other one `frontend_test.yml` and split tests a bit more cleanly